src/enums/endpoints.js

```diff
@@ -71,7 +71,10 @@ const FspEndpointTemplates = {
   ORACLE_PARTICIPANTS_BATCH: '/participants',
   TRANSFERS_POST: '/transfers',
   TRANSFERS_PUT: '{{fsp}}/transfers/{{ID}}',
-  TRANSFERS_PUT_ERROR: '/{{fsp}}/transfers/{{ID}}/error'
+  TRANSFERS_PUT_ERROR: '/{{fsp}}/transfers/{{ID}}/error',
+  BULK_TRANSFERS_POST: '/bulkTransfers',
+  BULK_TRANSFERS_PUT: '{{fsp}}/bulkTransfers/{{ID}}',
```

Comment:
I don't like the idea of this being capital case: {{ID}}. It stands for identifier, while fsp stands for Financial Services Provider. Please, change to {{id}}

Reply:
@ggrg Wouldn't that break the convention used in the surrounding code? Take a look at the whole file.

Reply:
I'm afraid it will also need to change. Of course it needs to be confirmed that it does not break working code. I don't agree that if this enum has been started wrongly, it should be continued because of that.

Reply:
@ggrg @oderayi this will require a change in all APIs that use the endpoint templates so if you are going to choose a standard please ensure that the changes are made to all applicable APIs i.e. ALS, ML-API, maybe Quoting

Reply:
Guys, if its that big, then please create a separate issue for it, but please let this PR through without it.. Lets not alter the scope of this PR drastically.

Reply:
@elnyry I agree

Reply:
Sure. I'm ok with the changes. Just wanted to mention that I recently has changed code and filenames in quoting-service because of capital {ID}.js files. This is an issue that we need to be strict about and implement throughout the code base, in order to avoid related issues.

Reply:
Thanks Georgi, Rajiv

Reply:
@ggrg the reason for those is that the API Swagger definition has it in uppercase, therefore it is created in that way. There is no other reason than that

Reply:
Thanks everyone.